emission/net/ext_service/otp/test_otp.py

```diff
@@ -0,0 +1,144 @@
+import unittest
```

Comment:
tests should be in `emission/tests` so that they can be included in the automated tests